diff -urN grass-6.4.2/gui/wxpython/Makefile grass-6.4.2-patched/gui/wxpython/Makefile
--- grass-6.4.2/gui/wxpython/Makefile	2011-04-13 23:57:55.000000000 +1100
+++ grass-6.4.2-patched/gui/wxpython/Makefile	2012-11-18 01:56:14.018911601 +1100
@@ -11,7 +11,7 @@
 ETCDIR = $(ETC)/wxpython
 
 SRCFILES := $(wildcard compat/* gui_modules/* icons/*.* icons/silk/* xml/*) gis_set.py wxgui.py README
-DSTFILES := $(patsubst %,$(ETCDIR)/%,$(SRCFILES)) $(patsubst %.py,$(ETCDIR)/%.pyc,$(filter %.py,$(SRCFILES)))
+DSTFILES := $(patsubst %,$(ETCDIR)/%,$(SRCFILES))
 
 default: install_scripts
 	$(MAKE) parsubdirs
